### ISSUE TITLE
Loot tables not working.

### ISSUE THREAD

**Issue by Subthy** [JMilamber/Brass_Amber_BattleTowers#60]
So, loot tables are not working for all levels except roof. So I tried to customize loot tables for the tower. The roof one worked perfectly, but the other ones from 1-8 aren't working at all. By the looks of it, it's using a built-in loot table that doesn’t change even when you add you’re own one.

In the file, I have added the roof loot table that works fine. I copied pasted it in all others to see it work, but it didn’t.
[roof.json.zip](https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/files/8053132/roof.json.zip)

**Comment by JMilamber** [comment]
Hi BigBoi!
Here are a couple of ideas/things to check -apologies if you have already done some of them, just covering all my bases!

I see that you have added a .json file, am I  assuming that you are editing the LootTable files directly and not the structure files for the mod?
Did you changed the name of any of the LootTables when you edited them or are they still named 1.json 2.josn etc?
When you are adding the new changed LootTable files, where are you putting them? like are you editing the ones in the mod jar itself or have you created a separate Datapack that you are loading on top of the mod?
Have you checked to see if copy/pasting the text of the LootTables into another mods LootTable works?
Finally, could you upload one or two of the LootTables you have tried that haven't worked for you?
.
Realistically, (as far as I know) there isn't much that can be wrong with LootTables, 
Either they are being put in the wrong spot, they have the wrong name, or the internals of the LootTable are not correct and Minecraft can't read it in.

**Comment by JMilamber** [comment]
Also be aware- If you change the loot tables for the 5/6/7/8 floors make sure you add the "ba_bt:monolith_key_land" into whatever LootTable you've constructed or you won't be able to unlock the Monolith and fight the Golem.

**Comment by Subthy** [comment]
Hello,

I am editing the LootTablr file directly and using a mod called Global Data- & Resourcepacks.
The loot tables are still called from 1-8 and roof. I copied pasted the roof loot table into the other ones, so I don't think there is a point in sending the other ones, as the contents will be the same as the roof loot table and yes I added the ba_bt:monolith_key_land. The roof.json is the only loot table working and the bellow ones are not. I even checked the bottom one, if 8 acted as the first floor.

**Comment by JMilamber** [comment]
[BT-Loot.zip](https://github.com/BrassAmber-Mods/Brass_Amber_BattleTowers/files/8107640/BT-Loot.zip)

Gonna be honest, the one time I tried to add the mod your using to test it, forge crashed. 
I honestly don't know what could be causing this, but above I have uploaded a datapack zip that contains all of the loot_table files. try editing the tables in this zip and then placing the whole zip folder into the folder created by the mod (if I understood how the mod works correctly) 

My only guess is that maybe the mod your using needs a full datapack with the entire data folder structure that BT has inside of it for it to correctly override the loot_tables. 

P.S. sorry it took me so long to respond. 
Loot Tables are not my Forte. If this idea fails I would recommend contacting the mod owner, they may have a better idea of whether it is a porblem with BT or something else. 
best of luck
